### PR TITLE
Added Google's ErrorProne compile time checker.

### PR DESCRIPTION
**Pull request by levonk**
Google's ErrorProne is very similar to FindBugs but it catches a few of the errors that aren't native FindBugs plugins.   It is also very actively developed.

### CODE REVIEW

**Comment by jon-ruckwood** [comment]
I like this. It's very easy to get started with. :+1:

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 *Tools that provide metrics and quality measurements.*
 
 * [Checkstyle](http://checkstyle.sourceforge.net/) - Static analysis of coding conventions and standards.
+* [ErrorProne](https://github.com/google/error-prone) - Catch common Java mistakes as compile-time errors from Google.
```
Comment (akullpp) [inline README.md]: The description is kind of misleading. You wanted to say that Google made the library but it sounds like it catches "Java  errors coming from Google"

Comment (tobijdc) [inline README.md]: To me the description sounds correct. It is what they describe it them selfs. They catch mistakes which go beyond compiler warnings and halt the build if they are severe. See example [SelfAssignment](http://errorprone.info/bugpattern/SelfAssignment)

Comment (akullpp) [inline README.md]: You misunderstood the issue here. The preposition *from* indicates that these errors are a result of some code Google wrote. Please fix the wording and the spelling of the library.

**Comment by akullpp** [comment]
Closed due to inactivity.

**Pull request closed without merging**